src/App.js

```diff
@@ -64,30 +73,16 @@ const App = () => {
     initializeState();
   }, [dispatch]);
 
-  const SnackbarIcon = <IconAlert className={classes.snackbarIconVariant} />;
-
   return (
     <SnackbarProvider
-      action={(notificationId) => (
-        // eslint-disable-next-line react/display-name
-        <IconButton color="inherit" onClick={onClickDismiss(notificationId)}>
-          <IconClose fontSize="small" />
-        </IconButton>
-      )}
+      className={classes.snackbarProvider}
       classes={{
         variantSuccess: classes.snackbarSuccess,
         variantError: classes.snackbarError,
         variantWarning: classes.snackbarWarning,
         variantInfo: classes.snackbarInfo,
       }}
-      iconVariant={{
-        info: SnackbarIcon,
-        default: SnackbarIcon,
-        error: SnackbarIcon,
-        success: SnackbarIcon,
-        warning: SnackbarIcon,
```

Comment:
Because we have different icons for the same type of notification and that is possible only in new version of the library which we currently cannot update due to old version of material design